composer.json

```diff
@@ -51,24 +51,23 @@
         "nikic/php-parser": "^4.2.1",
         "ocramius/package-versions": "^1.2",
         "pimple/pimple": "^3.2",
-        "sebastian/diff": "^1.4 || ^2.0 || ^3.0",
-        "symfony/console": "^3.4 || ^4.0",
-        "symfony/filesystem": "^3.4 || ^4.0",
-        "symfony/finder": "^3.4 || ^4.0",
-        "symfony/process": "^3.4 || ^4.0",
-        "symfony/yaml": "^3.4 || ^4.0",
+        "sebastian/diff": "^3.0.2",
+        "symfony/console": "^3.4.29 || ^4.0",
+        "symfony/filesystem": "^3.4.29 || ^4.0",
+        "symfony/finder": "^3.4.29 || ^4.0",
+        "symfony/process": "^3.4.29 || ^4.0",
+        "symfony/yaml": "^3.4.29 || ^4.0",
         "thecodingmachine/safe": "^0.1.16",
         "webmozart/assert": "^1.3",
         "webmozart/path-util": "^2.3"
     },
     "conflict": {
-        "symfony/console": "=3.4.16 || =4.1.5",
-        "symfony/process": "3.4.2"
+        "symfony/console": "=4.1.5"
```

Comment:
why is that one necessary?

Reply:
4.1.5 has an issue that throwing an exception still causes you to exit with a 0 code.

Reply:
That's #523